packages/labs/gen-wrapper-vue/goldens/test-element-a/src/ElementA.vue

```diff
@@ -1,27 +1,48 @@
 <script setup lang="ts">
-import {h, useSlots} from 'vue';
+import {h, useSlots, reactive} from 'vue';
 import {assignSlotNodes, Slots} from '@lit-labs/vue-utils/wrapper-utils.js';
 import '@lit-internal/test-element-a/element-a.js';
 
-const props = defineProps<{
+export interface Props {
   foo?: string | undefined;
-}>();
+}
+
+const vueProps = defineProps<Props>();
+
+const defaults = reactive({} as Props);
+const vDefaults = {
```

Comment:
It might be nice to see if all of this stuff (and the render stuff) could be pulled out to `vue/wrapper-utils.js` via some sort of composition API function thing that gets passed `vueProps`, since it doesn't contain any gencode itself... not sure it's possible.

Reply:
And somewhere, we need to add comments about the vDefaults gambit, since it's pretty needle-thready. Especially re: first render on client & SSR.